content/push/index.textile

```diff
@@ -78,4 +78,10 @@ Each push target device is associated with a unique @deviceId@ and authenticatio
 
 * By employing a standard "Ably key":/auth/basic or "Token":/auth/token, with a @deviceIdentityToken@, a credential generated during registration to assert the device's identity, included in the request header.
 
-The sevice credential management is handled by the "Ably SDK":https://ably.com/docs/sdks, removing the need for the client application to manage device credentials unless accessing the "push admin API":/api/realtime-sdk/push-admin directly via HTTP.
\ No newline at end of file
+The sevice credential management is handled by the "Ably SDK":https://ably.com/docs/sdks, removing the need for the client application to manage device credentials unless accessing the "push admin API":/api/realtime-sdk/push-admin directly via HTTP.
+
+h4(#Error). Error Handling
+
+The "metachannel":/metadata-stats/metadata/subscribe#metachannels @[meta]log:push@ handles errors in the push notification system, providing a dedicated stream for logging errors such as delivery failures, authentication issues, and configuration problems. 
```

Comment:
Have you been able to test the errors returned to the metachannel? Some of those mentioned (e.g. auth) I'd assume would be returned directly to the client itself. Similarly a configuration problem sounds like something that would error before reaching Ably.